pkg/controllers/user/authz/podsecuritypolicy/template.go

```diff
@@ -147,3 +155,25 @@ func GetPodSecurityPolicyTemplateID(projectLister v3.ProjectLister, clusterListe
 func KeyToPolicyName(key string) string {
 	return fmt.Sprintf("%v-psp", strings.Replace(key, "/", "-", -1))
 }
+
+func UpdatePolicyIfOutdated(templateLister v3.PodSecurityPolicyTemplateLister,
```

Comment:
Does this function need to be exported? Is it used outside of this package?

Reply:
No, there are a couple others that can be reduced in scope as well.  I'll make that change.